### DISCUSSION TITLE
IMPORTNAT

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6442] [General]
Add a button to restart ant media server , with confirmation and code channlege like 7+7 =   to restart ant media server

**Comment by asharamseervi** [comment]
Hi @brixxnatt,

Thank you for your suggestion and for raising this feature request. I believe this might be an opinionated feature request.

Adding a start, stop and restart button to the Management Console is an interesting idea while exploring & in-development with AMS. AMS relays more on CLI and APIs requests and while initially adopting AMS, few features might remain untapped by developers, adding more controls like these to the management console may add a better experience for beginners.

However, this could have significant implications, especially in production environments. It’s crucial to consider scenarios such as active/published streams, ongoing processes (like FFMpeg), and ensuring the user has administrator permissions.

@brixxnatt, Could you please share more about your specific use case? Understanding this will help the community and developers assess the impact and viability of your request.

Thank you!

**Comment by brixxnatt** [comment]
Hello , yes cause i need to restart the server daily after the streams , cause the RAM Stuck at 20 GB Rams  where there is 0 streams , so i need to  release the RAM , so login to the SSH and reboot the server ot reboot AMS everyday , untill MAS find a solution for this BUG .

**Reply by yashtandon113** [reply]
Hi @brixxnatt 
Could you please share the step-by-step reproduce scenario so that we can replicate the same issue on our end? In this way, we will be able to investigate properly.

Please include if you use any ABR as well.

**Comment by lastpeony** [comment]
After ram is stuck at 20 GB try to start streams. You should be able to start streams. The maximum number of streams server can handle should not change.
This occurs due to OS memory management.
